README.md

```diff
@@ -1,13 +1,23 @@
 # Project Mongo API
 
-Replace this readme with your own information about your project.
+✓ What MongoDB is
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+✓ How to model data in Mongo using Mongoose
+
```

Comment:
Very good and thorough readme.
You could edit the text regarding section nr 2 (The problem) to make it even better.